api/src/test/java/net/kyori/adventure/text/TextComponentTest.java

```diff
@@ -142,18 +142,6 @@ void testContains() {
     assertTrue(component.contains(child));
   }
 
-  // https://github.com/KyoriPowered/adventure/issues/363
```

Comment:
Don't remove this comment when you move this method to the new class.

Reply:
Do you want me to stick it at the top of the class? The test is kinda subsumed into all three of the new tests.

Reply:
sure